### PR TITLE
docs: add virtual hosts articles ADD and EDIT

### PR DESCRIPTION
**Pull request by souz4-my**
## Description of changes
Added articles describing how to add and edit a virtual host

## GitHub issues resolved by this PR
N/A

## Quality Assurance
- Once the changes in this PR are merged and deployed, success criteria is: The virtual host creation and edition steps on the documentation should reflect the current UX of Devopness

## More info
image

### CODE REVIEW

**Review by Diegiwg — APPROVED** [review]

File: docs/virtual-hosts/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Virtual Hosts
+intro: Learn how to configure virtual hosts to register domains and assign IPs for application access
```
Comment (jfoliveira) [inline docs/virtual-hosts/index.md:3]: Nope. At the momento Devopness does not `register domains`, so this text here is misleading to the users.

```suggestion
intro: Learn how to configure virtual hosts to have one or more applications publicly accessible on the internet by IP address or domain name
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/virtual-hosts/add-virtual-host.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: Add a Virtual host
+intro: Add a Virtual host to define how your applications accessible on the internet by IP address or domain name
```
Comment (jfoliveira) [inline docs/virtual-hosts/add-virtual-host.md:3]:
```suggestion
intro: Add a Virtual Host to make an application accessible on the internet, by server IP address or a domain name
```

File: docs/virtual-hosts/edit-virtual-host.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Edit a Virtual host
+intro: Edit a Virtual host to change how your applications are accessible on the internet by IP address or domain name
```
Comment (jfoliveira) [inline docs/virtual-hosts/edit-virtual-host.md:3]:
```suggestion
intro: Edit a Virtual Host to change how an application is accessible on the internet , by server IP address or a domain name
```

File: docs/virtual-hosts/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+title: Virtual Hosts
+intro: Learn how to configure virtual hosts to have one or more applications publicly accessible on the internet by IP address or domain name
```
Comment (jfoliveira) [inline docs/virtual-hosts/index.md:3]:
```suggestion
intro: Learn how to configure Virtual Hosts to have one or more applications accessible on the internet, by server IP address or a domain name
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

**Review by jfoliveira — APPROVED** [review]

**Review by thlmenezes — APPROVED** [review]

**Pull request merged**